src/wp-includes/rest-api/endpoints/class-wp-rest-themes-controller.php

```diff
@@ -447,11 +456,21 @@ public function get_item_schema() {
 					'type'        => 'string',
 					'readonly'    => true,
 				),
+				'stylesheet_uri' => array(
+					'description' => __( 'The uri for the theme\'s stylesheet directory.' ),
+					'type'        => 'string',
```

Comment:
We can add a `format` keyword with `uri`.

Reply:
Thanks, added in https://github.com/WordPress/wordpress-develop/pull/6399/commits/b1dd7ff35e3da3adc786ded589bd99d66bcd643e